packages/website/bin/page-generator/index.js

```diff
@@ -40,6 +41,15 @@ function generatePackagePages(packageInfo, generatorConfig) {
       titleCase(pkg.id),
     );
 
+    const changelogPath = path.join(homePath, 'changelog');
+    generateChangelogPage(
+      path.join(changelogPath, 'index.js'),
```

Comment:
Mostly curiousity;

```
path.join(changelogPath, 'index.js')
```
and
```
`changelogPath${'.js'}`
```

Are equivalent right? We're never going to include anything else under the changelog folder (I don't think) so is there a reason to use a folder index instead of a file?